utils/notification_service.py

```diff
@@ -314,7 +314,7 @@ def compute_diff_for_failure_reports(self, curr_failure_report, prev_failure_rep
         return entries_changed
 
     @property
-    def model_failures(self) -> Dict:
+    def model_failures(self) -> List[Dict]:
```

Comment:
Just fix the return type (forgot to update when I made change this block before)